pkgs/development/python-modules/flask-migrate/default.nix

```diff
@@ -1,25 +1,47 @@
-{ lib, buildPythonPackage, fetchPypi, isPy3k, glibcLocales, flask, flask_sqlalchemy, flask_script, alembic }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, alembic
+, flask
+, flask_script
+, flask_sqlalchemy
+, python
+}:
 
 buildPythonPackage rec {
   pname = "Flask-Migrate";
   version = "3.1.0";
+  format = "setuptools";
```

Comment:
Isn't this already the default?

Reply:
We plan to migrate away from a default format in the future.